simple/scalable_ep.c

```diff
@@ -495,7 +497,7 @@ static int run(void)
 		goto out;
 
 	run_test();
-
+	ft_finalize(tx_ep[0], scq[0], rcq[0], remote_rx_addr[0]);
```

Comment:
Hang issue due to ft_finalize call. Need to have a local finalize method for these two cases.